docs/helm_installation.md

```diff
@@ -63,9 +63,19 @@ Insprd will be exposed via NGINX by default, so you just need to have NGINX inst
 
 To properly access the cluster via the Inspr CLI it is necessary to configure the hostname in your machine to comport the Ingress host.
 
-This can be done in unix machines editting the file `etc/hosts` and adding `<cluster_IP> <insprIngress.host>`. For further understanding of what is happening access this [link](https://debian-handbook.info/browse/stable/sect.hostname-name-service.html)
+This can be done by installing the `insprctl`([example](cli_install.md)) and running the `init` command.
```

Comment:
It is not actually necessary to set the Hosts, you could use inpr withoud doing so. It would be ideal here to explain what situations the may want to set a specific host. @nicholaspcr can help you understand these different situations

Reply:
In simple terms the possibilities are the following:
- You have a URL that points to your cluster IP, therefore you can just set `serverip` to the url
- you have a cluster with an exposed IP but no URL pointing to it, just the kubernetes ingress that points to a service if someone gives a certain `host` to that IP. Therefore there is two simple ways of adding that `host` to the request:
    - Use the insprctl flag or config to set a `host` that will be added to the request
    - Add the `host` in your `/etc/hosts`, where you specify the IP and the hostname. This should not be recommended by us since it can be a pain for the user and its outside our application scope.


In my opinion would be nice to specify the URL pointing to the external IP of a cluster and just a cluster with the ingress. For a better understanding one could read the ingress documentation -> https://kubernetes.io/docs/concepts/services-networking/ingress/

Reply:
i changed my mind on this, if we consider that the next story is about the helm charts documentation the file itself has no need to exist.

Reply:
I agree with Nicholas, after some discussion with him and Pedro.